test/unit/app/data-fetchers/tequilapi-proposal-fetcher.spec.js

```diff
@@ -113,16 +114,18 @@ describe('TequilapiProposalFetcher', () => {
 
       expect(proposals.length).to.equal(2)
       expect(proposals[0]).to.deep.equal({
-        id: '0x1',
+        id: 1,
         providerId: '0x1',
         serviceType: 'openvpn',
-        serviceDefinition: {}
+        serviceDefinition: {},
+        metrics: undefined
       })
       expect(proposals[1]).to.deep.equal({
-        id: '0x2',
+        id: 2,
```

Comment:
TIL: these ids are not globally unique - unique proposal id is `id`+`providerId`+`serviceType`.
Would suggest to leave this id non-unique intentionally to make this more clear.

```suggestion
      id: 1,
```